_docs/protocols/2017-01-06-binary.md

```diff
@@ -222,9 +222,9 @@ Example:
 // TODO:
 ~~~
 ghci> hexEncode $ messageName (Proxy :: Proxy SendProxySK)
-
+"0102"
 ghci> hexEncode $ messageName (Proxy :: Proxy (DataMsg GtMsgContents))
-
+"020a03"
```

Comment:
Yes, I see, that it's `DataMsg`. But `DataMsg` is

```haskell
-- | Data message. Can be used to send actual data.
data DataMsg contents = DataMsg
    { dmContents :: !contents
    } deriving (Show, Eq)
```
according to docs. I don't understand where redundant bytes came from.